catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java

```diff
@@ -327,15 +330,22 @@ public boolean onContextItemSelected(MenuItem item) {
 
 	@Override
 	public void onSpriteChecked() {
+		Log.d("Lausi", "ONSpriteChecked");
```

Comment:
Erm still this does not explain the comment, as @LauraBebek mentioned Crowdin. In order to change a string at Crowdin, the `strings.xml` has to be modified, which in this case did not happen.

Btw: "Lausi" is not a good tagname :stuck_out_tongue: , more precisely this line should be removed